tox.ini

```diff
@@ -46,10 +46,10 @@ deps=
 
 [pytest_multi]
 deps=
-    pytest==6.2.5
-    pytest-randomly==3.8.0
-    pytest-cov==2.12.1
-    pytest-asyncio==0.15.1
+    pytest==7.0.0
+    pytest-randomly==3.11.0
+    pytest-cov==3.0.0
+    pytest-asyncio==0.18.0
```

Comment:
@DavidMinarsch Looks like the issues caused by the usage of `pytest==7.0.0` were some how fixed (maybe some code change on the packages)

Reply:
I'll push one more commit to unify these dependencies